public/content/de/basics/memory.md

```diff
@@ -0,0 +1,63 @@
+# Speicher
+
+D ist eine Systemprogrammiersprache und erlaubt daher manuellen Zugriff auf den Systemspeicher.
+Manueller Zugriff impliziert jedoch, dass Fehler sehr leicht entstehen können, daher
+verwendet D standardmäßig einen *Garbage Collector* um Speicher freizugeben.
+
+D verfügt über Adressstypen (aka Pointer) `T*` wie in C:
+
+    int a;
+    int* b = &a; // b beinhaltet die Adresse zu a
+    auto c = &a; // c hat den Typp int* und beinhaltet die Adresse zu a
+
+Mit dem `new` Ausdruck kann ein neuer Speicherblock auf dem Heap alloziert
+werden. Der `new` Ausdruck gibt einen Pointer zu den allozierten Speicher zurück:
+
+    int* a = new int;
+
+Sobald der Speicher, welcher durch `a` adressiert wird, von keiner Variable des Programmes mehr referenziert wird,
+wird der Speicher durch den Garbage Collector freigegeben.
+
+D erlaubt auch Adressarithmetik (`@system` Annotation), außer in Code der als `@safe` markiert wurde:
```

Comment:
@RalphBariz I would remove the "(`@system` Annotation)" here, because
- it's most likely confusing for the reader
- it will be explained below in more detail 
- there's `@trusted` too